packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -103,7 +100,7 @@ This source type can be used to build code from a BitBucket repository.
 
 ## Environment
 
-By default, projects use a small instance with an Ubuntu 14.04 image. You
+By default, projects use a small instance with an Ubuntu 18.04 image. You
```

Comment:
If you want to update this, you will also need to update:

https://github.com/awslabs/aws-cdk/blob/huijbers/bump-jsii/packages/@aws-cdk/aws-codebuild/lib/project.ts#L944
https://github.com/awslabs/aws-cdk/blob/huijbers/bump-jsii/packages/@aws-cdk/aws-codebuild/lib/project.ts#L625

And add a new image here:

https://github.com/awslabs/aws-cdk/blob/huijbers/bump-jsii/packages/@aws-cdk/aws-codebuild/lib/project.ts#L1026

Reply:
Sorry for not mentioning that in the PR. I have already pushed a commit to change the image, but forgot to update the documentation in the previous commit.

https://github.com/awslabs/aws-cdk/commit/87b1ea00a280157037b11a8bd7e580d7849ccd4a